site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -65,54 +68,320 @@ LAST SEEN   TYPE      REASON              OBJECT
+| NGINX errors                 | Reload failures on NGINX               | Fix permissions for control plane.                                                                                  | Security context not configured.                                                                                                        |
+| Usage reporting              | Errors logs related to usage reporting | Enable usage reporting. Refer to [Usage Reporting]({{< relref "installation/usage-reporting.md" >}})                | Usage reporting disabled.                                                                                                               |
+| Client Settings              | Request entity too large error         | Adjust client settings. Refer to [Client Settings Policy]({{< relref "../traffic-management/client-settings.md" >}})   | Payload is greater than the set [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size) |
+{{< /bootstrap-table >}}
+
+##### NGINX fails to reload
+
+NGINX reload errors can occur for various reasons, including syntax errors in configuration files, permission issues, and more. To determine if NGINX has failed to reload, check logs for your _nginx-gateway_ and _nginx_ containers.
+You will see the following error in the _nginx-gateway_  logs: `failed to reload NGINX:`, followed by the reason for the failure. Similarly, error logs in _nginx_ container start with `emerg`. For example, `2024/06/12 14:25:11 [emerg] 12345#0: open() "/var/run/nginx.pid" failed (13: Permission denied)` shows a critical error, such as a permission problem preventing NGINX from accessing necessary files.
+
+To debug why your reload has failed, start with verifying the syntax of your configuration files by opening a shell in the NGINX container following these [steps](#get-shell-access-to-nginx-container) and running `nginx -T`. If there are errors in your configuration file, the reload will fail and specify the reason for it.
+
+##### NGINX Gateway Fabric Pod is not running or ready
+
+To understand why the NGINX Gateway Fabric Pod has not started running or is not ready, the first step is to check the state of the Pod to get detailed information about the current status and events happening in the Pod. To do this, use `kubectl describe`:
+
+```shell
+kubectl describe pod <ngf-pod-name> -n nginx-gateway
+```
+
+The Pod description includes details about the image name, tags, current status, and environment variables. Verify that these details match your setup and cross-check with the events to ensure everything is functioning as expected. For example, the Pod below has two containers that are running and the events reflect the same.
```

Comment:
In the first paragraph we say "the first step is to...", should we include more steps after this?

Reply:
I see what that can be confusing. Meant it more like `first thing to do` and then more details on how to understand the `description` of the resource or what it means. I will add some more information on there and change the language.